vdslib/src/vespa/vdslib/state/nodestate.cpp

```diff
@@ -56,25 +55,24 @@ NodeState::NodeState(vespalib::stringref serialized, const NodeType* type)
       _capacity(1.0),
       _initProgress(0.0),
       _minUsedBits(16),
-      _startTimestamp(0)
+      _startTimestamp()
 {
 
     vespalib::StringTokenizer st(serialized, " \t\f\r\n");
     st.removeEmptyTokens();
-    for (vespalib::StringTokenizer::Iterator it = st.begin();
-         it != st.end(); ++it)
+    for (auto it : st)
```

Comment:
Consider using a non-generic name for loop variable (`token`?)

Additional nit: consider moving brace on next line up onto this line since this is now a single-line statement

Reply:
Thanks, will fix when reapplying